docs-kits/kits/knowledge-agents/development-view/api.md

```diff
@@ -214,7 +214,7 @@ SERVICE ?oemConnector {
                     ].           
 
     ?tieraConnector cx:offersAsset ?prognosisAsset.
-    ?prognosisAsset rdfs:isDefinedBy <https://raw.githubusercontent.com/catenax-ng/product-knowledge/main/ontology/prognosis_ontology.ttl>.
+    ?prognosisAsset rdfs:isDefinedBy <https://raw.githubusercontent.com/big-data-spaces/knowledge-agents-deployment/main/ontology/prognosis_ontology.ttl>.
```

Comment:
This link don't seem to work -> 404 error

Reply:
thanks for the heads up. These links were stale as we
- moved the ontology source.
- established an official w3id.org namespace redirection (for being independent of the actual source location).